src/portfinder/portfind.py

```
@@ -47,6 +47,8 @@ async def async_port_find(address, start, end, timeout):
 
     if address == "localhost":
         address = LOCAL_HOST_ADDRESS
+    else:
+        address = gethostbyname(address)
```

Ruff (0.12.2)

Use ternary operator `address = LOCAL_HOST_ADDRESS if address == "localhost" else gethostbyname(address)` instead of `if`-`else`-block

Replace `if`-`else`-block with `address = LOCAL_HOST_ADDRESS if address == "localhost" else gethostbyname(address)`

(SIM108)